axis.go

```diff
@@ -451,41 +451,70 @@ func (ts ConstantTicks) Ticks(float64, float64) []Tick {
 	return ts
 }
 
-// UnixTimeTicks is suitable for axes representing time values.
-// UnixTimeTicks expects values in Unix time seconds.
-type UnixTimeTicks struct {
+// TimeTicks is suitable for axes representing time values.
+// By default, TimeTicks expects values in (UTC) Unix time seconds.
```

Comment:
Unix time represents moment in time. It is time zone independent, so _(UTC) Unix time seconds_ doesn't make sense.